src/web/templates/faq.html

```diff
@@ -787,22 +787,10 @@ <h2 class="accordion-header" id="mtpwindows">
             </div>
           </div>
         </div>
-        <div class="accordion-item">
```

Comment:
so this way the question doesn't follow the "accordion" pattern that everything else does.
But maybe that's fine